### PR TITLE
feat(content): Edits to Remnant: Logistics 1

### PR DESCRIPTION
**Pull request by roadrunner56**
**Content (Artwork / Missions / Jobs)**

## Summary
Minor edits and additions to the Remnant: Logistics 1 mission. I changed the `to offer` conditions so that it would only show up after you had heard the phrase 'the Embers' used metaphorically by the Remnant, as in 'may the Embers burn bright for you'. When I played this mission I had not gotten very far through the Remnant storyline and my character would be unaware of the significance of that saying in Remnant culture, so I thought it was odd that I could sing about the Embers despite this. To fix this I locked this mission behind the first mission I could find that mentions the Embers in a metaphorical way.
In addition, I decided to add a few more options for the player when responding to the woman working at the information desk, with the choice of signing, singing, or talking to her. I thought this would be more engaging and give the player some more roleplaying opportunities, while adding some easter eggs about Remnant culture.

## Testing Done
TBD

## Save File
TBD

### CODE REVIEW

File: data/remnant/remnant 1 introduction.txt [hunk 1]
```diff
@@ -3821,12 +3821,29 @@ mission "Remnant: Logistics 1"
...
+					has "event: remnant: sign studies complete"
+				(Talk in reply.)
+			`You clear your throat. "Yes, and I believe someone requested that I come here."`
+				goto shesings
+			label sign
+			`You begin to sign, "Yes, and I believe someone requested that I come here."`
+			`	Her face lights up as she realizes that you are signing in her language. "I am happy to see that you have learned our language, but I can tell that it comes with considerable effort to you," she sings in a soft voice. "I will use the ancestral tongue in order to make sure my message is clear to you."`
+				goto shesings
+			label sing
+			`You respond by singing, "The Embers guided me to friends I didn't know, and your message invited me to meet a new one."`
```
Comment (TheGiraffe3) [inline data/remnant/remnant 1 introduction.txt:3843]:
```suggestion
			`	You respond by singing, "The Embers guided me to friends I didn't know, and your message invited me to meet a new one."`
```

File: data/remnant/remnant 1 introduction.txt [hunk 2]
```diff
@@ -3821,12 +3821,29 @@ mission "Remnant: Logistics 1"
...
+				(Sing in reply.)
+				goto sing
+				(Sign in reply.)
+				to display
+					has "event: remnant: sign studies complete"
+				(Talk in reply.)
+			`You clear your throat. "Yes, and I believe someone requested that I come here."`
+				goto shesings
+			label sign
+			`You begin to sign, "Yes, and I believe someone requested that I come here."`
```
Comment (TheGiraffe3) [inline data/remnant/remnant 1 introduction.txt:3839]:
```suggestion
			`	You begin to sign, "Yes, and I believe someone requested that I come here."`
```

File: data/remnant/remnant 1 introduction.txt [hunk 3]
```diff
@@ -3821,12 +3821,29 @@ mission "Remnant: Logistics 1"
...
-			`	Her hands make a short series of shakes and a moment later she remembers to laugh for you. The delayed response makes her laugh again, spontaneously this time, and you both join in. When she recovers herself, she sings, "Logistics have requested that we find someone who can carry <tons> of cargo to <planet>. Is the <ship> available immediately?"`
+			`	The woman working the desk seems startled to see you when you walk up. You can see the realization dawn on her as she looks you over with a practiced eye. "Ah," she sings, "you are the ancestral sojourner among us." You are by now used to the Remnant gawking at you.`
+			choice
+				`	(Sing in reply.)`
+					goto sing
+				`	(Sign in reply.)`
+					to display
+						has "event: remnant: sign studies complete"
+				`	(Talk in reply.)`
+			`You clear your throat. "Yes, and I believe someone requested that I come here."`
```
Comment (TheGiraffe3) [inline data/remnant/remnant 1 introduction.txt:3836]:
```suggestion
			`	You clear your throat. "Yes, and I believe someone requested that I come here."`
```

File: data/remnant/remnant 1 introduction.txt [hunk 4]
```diff
@@ -3821,12 +3821,29 @@ mission "Remnant: Logistics 1"
 	to offer
 		has "remnant: blood test pure"
 		has "Remnant: Defense 1: done"
+		has "Remnant: Technology Available: offered"
```
Comment (warp-core) [inline data/remnant/remnant 1 introduction.txt:3824]: The Technology Available mission requires all the missions in the Defense chain, and so we don't also need to check for that.
```suggestion
		has "Remnant: Technology Available: offered"
```

**Review by warp-core — COMMENTED** [review]
Furhter down in the "on offer" conversation, there is another line that assumes the player doesn't know sign language. "You manage to chant and point your way through the cafeteria line," etc.
The original offer conditions make it fairly unlikely that the player would encounter this after learning sign language, but the new conditions make it less so.

I'm not sure not offering this until after the player has acquired the Remnant License makes sense, given the content of the "on complete" conversation. The level of trust imparted in "Remnant: Technology Available" feels significantly beyond what is discussed in this mission.
Additionally, this unlocks some transport jobs, and I think it makes sense to have them available sooner rather than later, so that the jobs associated with this mission are available alongside the bounty jobs the player needs to do.

It might be best to remove the reference to the phrase the player probably hasn't heard yet, rather than try to move this mission to a later point.

**Comment by roadrunner56** [comment]
>It might be best to remove the reference to the phrase the player probably hasn't heard yet, rather than try to move this mission to a later point.

This is a good point and probably the better solution. I'll close this PR and try to implement this solution instead.

**Pull request closed without merging**